tests/unit/modules/network/vyos/fixtures/vyos_firewall_rules_config_v14.cfg

```diff
@@ -0,0 +1,24 @@
+set firewall ipv4 name V4-INGRESS default-action 'accept'
```

Comment:
Hi, I have tested this module on my vyos instance and I found minor bug. 

When using log in rule set, the collection renders this command

```
set firewall ipv4 name LAN-LAB rule 999 log 'enable'
```

Which is not valid for my vyos 1.4. I tried to find where/how to fix it but this python code is to complicated for me. Thanks for your hard work

Reply:
The `state` and `log` items should be fixed for 1.4+